extensions/hadoop/src/main/java/com/hazelcast/jet/hadoop/HadoopSources.java

```diff
@@ -62,12 +62,30 @@
      *
      * <pre>{@code
      *     Configuration conf = new Configuration();
-     *     conf.set(HadoopSources.COPY_ON_READ, "false");
+     *     conf.setBoolean(HadoopSources.COPY_ON_READ, false);
      *     BatchSource<Entry<K, V>> source = HadoopSources.inputFormat(conf);
      * }</pre>
      */
     public static final String COPY_ON_READ = "jet.source.copyonread";
 
+    /**
+     * When reading files from local file system using Hadoop, each processor
+     * reads files from its own local file system. If the local file system
+     * is shared between members, e.g multiple members on a machine, you should
```

Comment:
AFAIK we don't recommend running multiple members on a single machine, so the javadoc should give a better example, e.g. NFS mounted filesystem or similar.

Reply:
👍